docs/conf.py

```diff
@@ -695,7 +695,7 @@
 # Defaults to READTHEDOCS env variable, typically "/en/latest/"
 # Note special case when using default GitHub pages URL: "/<repo>/"
 # https://sphinx-notfound-page.readthedocs.io/en/latest/faq.html#does-this-extension-work-with-github-pages
-notfound_urls_prefix = "/yasa/"
+notfound_urls_prefix = None
```

Comment:
The 404 page is not formatting correctly. There are two things related to the 404 page, just as last time: (1) 404 page itself does not format correctly, (2) the header links on the 404 page do not compile correctly, they add a yasa suffix despite this variable change, eg `yasa-sleep.org/yasa/api.html`. So it's bad because people hit the 404 page and basically can't link out. I'm looking into it now, should be solveable soon!

Reply:
but good news is that all the old links are redirecting properly to `yasa-sleep`! So this fix is only for cases where people use _genuinely_ broken links to hit the 404.

Reply:
wait -- @raphaelvallat I'm confused and maybe spoke too early. Just checked the GH Action logs and I don't see a docs build for this PR, so is the live site a result of the build from this code?

Reply:
(sorry, just seeing your edit. Yes I agree, let me make that snippet)

Reply:
@raphaelvallat I'm still confused about if this code was used to generate the current live site. The `docs.yml` Action only runs automatically from pushes to `master`, or did you run it manually in the Actions tab on GH? That's technically possible but I don't see a log of it. If not, we should do that to see what happens with this code before I start changing stuff.

Reply:
<img width="1749" alt="image" src="https://github.com/user-attachments/assets/23471fe7-88c5-459a-b5ec-ecfba080e669" />
 
I'm not able to manually run the workflow for this branch because of branch protection rules, but you're right, let's merge this PR to `master` and see what the live site looks like

Reply:
Odd! I just ran manually from my end and it worked... not sure how that could be, let's look into it, I don't have access to settings like that.